<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="MisTest"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="10.189">
  <properties />
            <testcase name="misHomecatTest" time="10.189" classname="com.mis.market.test.MisHomecatTest">
                <failure type="org.junit.ComparisonFailure"
                  message="expected:&lt;[保存成功!]&gt; but was:&lt;[品类推荐不满8个]&gt;"
              >
<![CDATA[org.junit.ComparisonFailure: expected:<[保存成功!]> but was:<[品类推荐不满8个]>
  org.junit.Assert.assertEquals(Assert.java:125)
  org.junit.Assert.assertEquals(Assert.java:147)
  com.mis.market.pages.MisHomecatPage.save_success(MisHomecatPage.java:74)
  com.mis.market.test.MisHomecatTest.misHomecatTest(MisHomecatTest.java:58)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.Surefire.run(Surefire.java:180)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
  org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
